components/community/Hero.tsx

```diff
@@ -0,0 +1,48 @@
+import orbitData from '../../config/orbitData.json';
+import Header from './Header';
+
+interface HeroProps {
+  className?: string;
+};
+
+/**
+ * @description This component displays Hero Section.
+ * @param {HeroProps} props - The props for Hero component.
+ * @param {string} props.className - Additional CSS classes for styling.
+ */
+export default function Hero({
+  className = ''
+}: HeroProps): React.ReactNode {
```

Comment:
```suggestion
}: HeroProps) {
```

Don't specify the return type of functional components, that are exported by default. It will be taken automatically by Typescript.

Reply:
Done